cosmic_ray/commands/report.py

```diff
@@ -1,15 +1,19 @@
 def _print_item(item):
-    return [
-        'job ID: {}'.format(item.work_id),
-        'module: {}'.format(item.module_name),
-        'operator: {}'.format(item.operator_name),
-        'occurrence: {}'.format(item.occurrence),
+    result = item.result_data
+    result_type = item.result_type
+    if result_type in ['normal', 'exception']:
+        result_type = result[1][0]
+    ret_val = [
+        'job ID {}:{}:{}'.format(
+                                item.work_id,
+                                result_type,
+                                item.module_name),
         'command: {}'.format(
             ' '.join(item.command)
             if item.command is not None else ''),
-        'result type: {}'.format(item.result_type),
-        'data: {}'.format(item.result_data)
         ]
+    ret_val.extend(result[1][1])
```

Comment:
It seems a bit strange to have this as an `extend()` rather than just adding it as part of the list literal above. Am I missing something?